python/ray/ray_perf.py

```diff
@@ -151,19 +151,22 @@ def actor_async():
 
     timeit("1:1 actor calls async", actor_async, 1000)
 
-    a = Actor.options(is_direct_call=True).remote()
+    a = Actor.options(max_concurrency=16).remote()
```

Comment:
Remove the flagged direct call microbenchmarks, they are now just normal microbenchmarks.